packages/cli/src/commands/start.ts

```diff
@@ -173,10 +173,10 @@ export class Start extends BaseCommand {
 		await super.init();
 		this.activeWorkflowRunner = Container.get(ActiveWorkflowRunner);
 
-		await this.initLicense();
```

Comment:
License needs to be init first because orchestration depends on the enterprise features, so we cannot put it afterwards.

Reply:
`License` depends on `OrchestrationService` for `onFeatureChange` and for the change in this PR. But `OrchestrationService` depends on `License` for checking if multi-main setup is licensed. Let me see how to untangle.